chart/kubedb/templates/deployment.yaml

```diff
@@ -36,8 +36,6 @@ spec:
         args:
         - run
         - --v=3
-        - --docker-registry={{ .Values.kubedb.registry }}
```

Comment:
Should be removed from chart values.yaml, README.md

Reply:
used in Operator Deployment. https://github.com/kubedb/cli/blob/add-catalog/chart/kubedb/templates/deployment.yaml#L34